README.md

```diff
@@ -63,15 +63,24 @@ Usage example:
         tune.exfat -l /dev/sda1
     2. set new volume label.
         tune.exfat -L "new label" /dev/sda1
+    3. print current volume serial.
+        tune.exfat -i /dev/sda1
+    4. set new volume serial.
+        tune.exfat -I "new serial" /dev/sda1
 
 - exfatlabel:
-    Get or Set volume label
+    Get or Set volume label or serial
 
 Usage example:
     1. get current volume label.
         exfatlabel /dev/sda1
     2. set new volume label.
-        exfatlabel "new label" /dev/sda1
+        exfatlabel /dev/sda1 "new label"
+    3. get current volume serial.
+        exfatlabel -i /dev/sda1
+    4. set new volume serial.
+        exfatlabel -i /dev/sda1 "new serial"
```

Comment:
And I wonder setting/getting volume serial should be implemented in exfatlabel. I think it is better to implement this in only tune.exfat.

Reply:
You can see fatlabel -i option. fatlabel can change volume id as well as volume label. The requester who requested this option want this option is addded to exfatlabel like fatlabel. tune.exfat will have more function included both option.

Reply:
Okay, I got it. Thank you!